app/views/smart_answers/show.html.erb

```diff
@@ -1,5 +1,13 @@
 <% content_for :title do %><%= @presenter.title %><% end %>
 <% content_for :head do %>
+  <script>
+    var SmartAnswer = SmartAnswer || {};
+    <% if ajax_enabled_for?(@presenter.name) %>
+      SmartAnswer.AJAX_ENABLED = true;
+    <% else %>
+      SmartAnswer.AJAX_ENABLED = false;
+    <% end %>
+  </script>
```

Comment:
@selfthinker 

You `ajax_enabled_for?(name)` is a predicate method returning booleans..

You could try

```javascript
  <script>
    var SmartAnswer = SmartAnswer || {};
    SmartAnswer.AJAX_ENABLED = <%= ajax_enabled_for?(@presenter.name) %>;
  </script>
```